platform/mock-radio.ts

```diff
@@ -58,16 +60,47 @@ export class MockNetwork {
   }
 }
 
-export class UDPDevice implements MockUDPListener {
-  private udpMessageAction: (msg: Buffer, rinfo: RemoteAddressInfo) => void;
+export class UDPDevice {
+  private deviceId: string;
+  private port: number;
+  private address: string;
+  private network: MockNetwork;
 
-  onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): void {
-    this.udpMessageAction(msg, rinfo);
+  public constructor(
+    deviceId: string,
+    network: MockNetwork,
+    port: number,
+    address: string
+  ) {
+    this.deviceId = deviceId;
+    this.network = network;
+    this.port = port;
+    this.address = address;
   }
 
-  setUDPMessageAction(
-    messageAction: (msg: Buffer, rinfo: RemoteAddressInfo) => void
-  ) {
-    this.udpMessageAction = messageAction;
+  public getDeviceId(): string {
+    return this.deviceId;
+  }
+
+  public getDevicePort(): number {
+    return this.port;
+  }
+
+  /**
+   * Sends a `discoveryBuffer` to the port specified in a `UDPScanConfig`
+   * @param discoveryBuffer
+   * @param scanConfig
+   */
+  public sendDiscoveryBuffer(
```

Comment:
Should we move this to the platform class and remove the scanConfig parameter? It's not really strictly tied to the radio simulation anymore.

Reply:
Yeah, that makes sense.  I also removed the reference to the `MockNetwork` and the `MockUDPListener` implementation for now since it's now not being used.  I anticipate needing it later for `EXECUTE` #16.